### ISSUE TITLE
benchmark: I used it for training my proposed model

### ISSUE THREAD

**Issue by AFAgarap** [zalandoresearch/fashion-mnist#8]
I proposed a GRU+SVM model at the university as my undergraduate research, the proposal paper may be read [here](https://www.researchgate.net/publication/317016806_A_Neural_Network_Architecture_Combining_Gated_Recurrent_Unit_GRU_and_Support_Vector_Machine_SVM_for_Intrusion_Detection). Simply put, my proposal is to use SVM as the classification function for GRU instead of the 'conventional' Softmax.

Here are the results of the training using your dataset:

```
Epoch : 0 completed out of 10, loss : 274.4726867675781, accuracy : 0.7890625
Epoch : 1 completed out of 10, loss : 201.513671875, accuracy : 0.87890625
Epoch : 2 completed out of 10, loss : 201.049072265625, accuracy : 0.859375
Epoch : 3 completed out of 10, loss : 155.28115844726562, accuracy : 0.890625
Epoch : 4 completed out of 10, loss : 145.94015502929688, accuracy : 0.9140625
Epoch : 5 completed out of 10, loss : 148.19613647460938, accuracy : 0.88671875
Epoch : 6 completed out of 10, loss : 155.27915954589844, accuracy : 0.87890625
Epoch : 7 completed out of 10, loss : 192.79263305664062, accuracy : 0.8671875
Epoch : 8 completed out of 10, loss : 151.52243041992188, accuracy : 0.90234375
Epoch : 9 completed out of 10, loss : 171.39292907714844, accuracy : 0.8984375
Accuracy : 0.8878000378608704
```

The source may be found in my [GitHub Gist](https://gist.githubusercontent.com/AFAgarap/92c1c4a5dd771999b0201ec0e7edfee0/raw/828fbda0e466dacb1fad66549e0e3022e1c7263a/gru_svm_zalando.py). The hyper-parameters used were as follows:

```
BATCH_SIZE = 256
CELL_SIZE = 256
EPOCHS = 10
LEARNING_RATE = 0.01
NUM_CLASSES = 10
SVM_C = 1
```
Trained using `tf.train.AdamOptimizer()`, and used `tf.nn.static_rnn()`.

**Comment by AFAgarap** [comment]
I did another training with the following hyper-parameters:

```
BATCH_SIZE = 256
CELL_SIZE = 256
DROPOUT_P_KEEP = 0.85
EPOCHS = 10
LEARNING_RATE = 1e-3
SVM_C = 1
```

Here's the result:

```
Epoch : 0 completed out of 10, loss : 176.0813446044922, accuracy : 0.875
Epoch : 1 completed out of 10, loss : 198.86962890625, accuracy : 0.84375
Epoch : 2 completed out of 10, loss : 160.88693237304688, accuracy : 0.87109375
Epoch : 3 completed out of 10, loss : 206.49061584472656, accuracy : 0.84765625
Epoch : 4 completed out of 10, loss : 169.35137939453125, accuracy : 0.84375
Epoch : 5 completed out of 10, loss : 158.23252868652344, accuracy : 0.8984375
Epoch : 6 completed out of 10, loss : 210.0270233154297, accuracy : 0.85546875
Epoch : 7 completed out of 10, loss : 173.9663848876953, accuracy : 0.875
Epoch : 8 completed out of 10, loss : 190.8488006591797, accuracy : 0.8671875
Epoch : 9 completed out of 10, loss : 161.12210083007812, accuracy : 0.89453125
Accuracy : 0.8551000356674194
```

The implementation I used for this test may be found [here](https://gist.githubusercontent.com/AFAgarap/92c1c4a5dd771999b0201ec0e7edfee0/raw/828fbda0e466dacb1fad66549e0e3022e1c7263a/gru_svm_zalando_dropout.py).
